pkg/util/fixed8.go

```diff
@@ -46,11 +46,17 @@ func (f Fixed8) FloatValue() float64 {
 	return float64(f) / decimals
 }
 
-// Int64Value returns the original value representing Fixed8 as int64.
-func (f Fixed8) Int64Value() int64 {
+// IntegralValue returns integer part of the original value representing
+// Fixed8 as int64.
+func (f Fixed8) IntegralValue() int64 {
 	return int64(f) / decimals
 }
 
+// FractionalValue returns decimal part of the original value.
```

Comment:
I think this needs some commend regarding negative numbers.
Like `It has the same sign as f, so that f = f.IntegralValue() + f.FractionalValue()`